autospotting.go

```diff
@@ -30,8 +30,15 @@ func main() {
 func run() {
 	log.Println("Starting autospotting agent, build:", Version)
 
-	log.Printf("Parsed command line flags: regions='%s' min_on_demand_number=%d min_on_demand_percentage=%.1f",
-		conf.Regions, conf.MinOnDemandNumber, conf.MinOnDemandPercentage)
+	log.Printf("Parsed command line flags: "+
+		"regions='%s' "+
+		"min_on_demand_number=%d "+
+		"min_on_demand_percentage=%.1f "+
+		"keep_instance_type=%t",
```

Comment:
I think with the new semantic a better name for this parameter would be something like "allowed_instance_types"